docs/config_8h.html

```diff
@@ -545,7 +545,8 @@ <h2 class="memtitle"><span class="permalink"><a href="#a359b781cc4fde8dd09e30b95
       </table>
 </div><div class="memdoc">
 
-<p>The string placed directly before the human-readable logging level. </p>
+<p>Define to use &lt;CR&gt;&lt;NL&gt; line endings, otherwise use &lt;NL&gt;. </p>
```

Comment:
hate to be that guy, but it should be `CRLF` and `LF` (since those are the official ASCII character short codes).

Are we using CRLF by default right now? I think the default should be LF if !defined(__WIN__) and CRLF if defined (w/ override available)

Reply:
Sure, I'll switch the name.  It was always using LF before, so I left that default, but I should say I never had any windows editor not like the logs or anything, honestly.  Not sure it really matters post DOS days?

Reply:
Fixed.